plugins/event_notification/providers/http/lib/KDispatchHttpNotificationEngine.class.php

```diff
@@ -182,13 +184,18 @@ public function sendHttpRequest(KalturaHttpNotificationTemplate $httpNotificatio
 		$errCode = $curlWrapper->getErrorNumber();
 		$errMessage = $curlWrapper->getError();
 
+		$header_size = curl_getinfo($curlWrapper->ch, CURLINFO_HEADER_SIZE);
+		$headers = substr($results, 0, $header_size);
+		$body = strip_tags(substr($results, $header_size));
+
 		$curlWrapper->close();
 
-		KalturaLog::info("HTTP Request httpCode [" . $httpCode . "] Results [$results]");
+        	KalturaLog::info("HTTP Request httpCode [" . $httpCode . "] Results [$results] Headers [$headers] Body [$body]");
 		if(!$results || $httpCode != 200)
 		{
+            $		body = substr($body, 0, 1000);
```

Comment:
you have tons of spaces between $ and body param

Reply:
Fixed. https://github.com/kaltura/server/pull/12088/commits/bd7bfa77710425308a8cdf27bccf007d16a4d987